src/main/java/core/basesyntax/service/impl/StorageServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.StorageService;
+import core.basesyntax.strategy.OperationCalculator;
+import core.basesyntax.strategy.OperationCalculatorStrategy;
+import java.util.List;
+import java.util.Map;
+
+public class StorageServiceImpl implements StorageService {
```

Comment:
Let's come up with a better name for this service. StorageService sounds like it will contain all logic connected with our storage. Like it can be anything. For example it can be `TransactionProcessingService`

Reply:
fixed